nbs/tsdataset.ipynb

```diff
@@ -54,7 +54,8 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #77.            temporal_cols = set(temporal_cols)
</pre><p>This should also include the target</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I've added the target at a later stage **temporal_cols = pd.Index([target_col] + list(exogs))** as later in the code there's the assumption that the target is the first index of temporal_cols. I can add it to the set too but I think I would then have to remove it and still call the above to make sure it is the first element in the list?